sql/core/src/test/scala/org/apache/spark/sql/ParametersSuite.scala

```diff
@@ -624,6 +624,67 @@ class ParametersSuite extends QueryTest with SharedSparkSession with PlanTest {
     comparePlans(expected, parameterizedSql)
   }
 
+  test("SPARK-49017: bind named parameters with IDENTIFIER clause") {
+    withTable("testtab") {
+      // Create table
+      spark.sql("create table testtab (id int, name string)")
+
+      // Insert into table using single param - WORKS
+      spark.sql("insert into identifier(:tab) values(1, 'test1')", Map("tab" -> "testtab"))
+
+      // Select from table using param - WORKS
+      checkAnswer(spark.sql("select * from identifier(:tab)", Map("tab" -> "testtab")),
+        Seq(Row(1, "test1")))
+
+      // Insert into table using multiple params - WORKS
+      spark.sql("insert into identifier(:tab) values(2, :name)",
+        Map("tab" -> "testtab", "name" -> "test2"))
+
+      // Select from table using param - WORKS
+      checkAnswer(spark.sql("select * from identifier(:tab)", Map("tab" -> "testtab")),
```

Comment:
This is a duplicated test as https://github.com/apache/spark/pull/47501/files#diff-509971ca229888a8895c9402f41a4ed82989e2678fff4036e9b6b03dfa253fdaR636

Reply:
Yeah, it is not necessary to do parameters here. Will change it to normal select, as the aim of this query was to check contents of the table after previous addition.